js/account-popover.jquery.js

```diff
@@ -175,6 +175,10 @@
             break;
           case 'layoutDidMount':
             $popover.addClass('loaded');
+            this.updateColourTheme(data.colourTheme);
```

Comment:
If you have an old version of the MyWarwick code (admittedly unlikely), could this lead to `data.colourTheme` being `undefined`?

Reply:
Yes, in which case it will use the default colour, which is what the theme would be anyway.